benchmarks/progress.py

```diff
@@ -67,3 +67,24 @@ def get_progress_since(date):
                 result.append(progress)
 
     return result
+
+
+def get_progress_betwewn_results(current, baseline):
+    result = []
+
+    for current_testjob in current.test_jobs.all():
+
+        environment = current_testjob.environment
+        baseline_testjob = baseline.testjobs.get(environment=environment)
+
+        if baseline_testjob:
+            progress = Progress(
+                project=current.name,
+                branch=current.branch_name,
+                environment=environment,
+                before=baseline_testjob,
+                after=current_testjob,
+            )
+            result.append(progress)
+
+    return result
```

Comment:
This _if_ is here to handle the case where a build B has test jobs for environments E1, E2, ..., En, but the baseline build does not have test job results for all of those environments. In this case, the environments that are not covered in the baseline would be ignored in the comparison.